### ISSUE TITLE
can't access to menu anymore on android M preview

### ISSUE THREAD

**Issue by kalon33** [CUTR-at-USF/OpenTripPlanner-for-Android#476]
Using android M preview, it is not possible to access the menu anymore, as menu softkey is not available.

That makes the app a lot less usable.

**Comment by barbeau** [comment]
Try pressing and holding the Multitasking button (next to Home).  I haven't
tried this on M preview but it works on earlier Android versions.

Either way, we should provide a better way to get to the menu.

**Comment by kalon33** [comment]
It doesn't seem to work on M. I have a second vibration when doing this, but nothing happen on release, apart the list of current apps.

**Comment by barbeau** [comment]
Yikes, that's not good.  Strange that they would break backwards capability for apps that haven't updated to the M SDK yet.  Is this on a Nexus 5?

**Comment by barbeau** [comment]
And which M preview / build version?

**Comment by kalon33** [comment]
It is on a Nexus 9 (tablet) with the third (latest) M preview.

**Comment by kalon33** [comment]
After upgrading to Android 6.0 final, I have the same problem.

**Comment by barbeau** [comment]
Ouch.  We'll definitely need to fix this and find another way for the user to open the menu.

@vreixo were you working on some design ideas for moving the settings at some point?  Any thoughts?

**Comment by barbeau** [comment]
@kalon33 Can you try long pressing the back button and see if that brings up the menu?  Apparently that's the solution on Note 5 (not Marshmallow):
http://forums.androidcentral.com/samsung-galaxy-note-5/569132-note-5-menu-button.html

**Comment by kalon33** [comment]
@barbeau I just tried it on my Nexus 9, but that doesn't trigger anything else than the default back button action.

----- Mail d'origine -----
De: Sean Barbeau <notifications@github.com>
À: CUTR-at-USF/OpenTripPlanner-for-Android <OpenTripPlanner-for-Android@noreply.github.com>
Cc: Nicolas Derive <kalon33@ubuntu.com>
Envoyé: Wed, 21 Oct 2015 04:21:55 +0200 (CEST)
Objet: Re: [OpenTripPlanner-for-Android] Can't access menu on devices without Menu key for M Preview 3 (#476)

@kalon33 Can you try long pressing the back button and see if that brings up the menu?  Apparently that's the solution on Note 5 (not Marshmallow):
http://forums.androidcentral.com/samsung-galaxy-note-5/569132-note-5-menu-button.html

---

**Comment by barbeau** [comment]
@kalon33 Finally had a chance to look at this closer - when I fire up an emulator with a Nexus 5 profile with Android 6.0, I do see a compatibility mode "3 dots" menu indicator in the bottom right corner of the screen, next to the "multitasking" button.  If I tap on that, I can open the menu:

![image](https://cloud.githubusercontent.com/assets/928045/11189816/3a9cc65c-8c60-11e5-81e8-5851fa69d96f.png)

Do you see this on your Nexus 9?

**Comment by kalon33** [comment]
@barbeau no, unfortunately not...
----- Mail d'origine -----
De: Sean Barbeau <notifications@github.com>
À: CUTR-at-USF/OpenTripPlanner-for-Android <OpenTripPlanner-for-Android@noreply.github.com>
Cc: Nicolas Derive <kalon33@ubuntu.com>
Envoyé: Mon, 16 Nov 2015 18:48:20 +0100 (CET)
Objet: Re: [OpenTripPlanner-for-Android] Can't access menu on devices without Menu key for M Preview 3 (#476)

@kalon33 Finally had a chance to look at this closer - when I fire up an emulator with a Nexus 5 profile with Android 6.0, I do see a compatibility mode "3 dots" menu indicator in the bottom right corner of the screen, next to the "multitasking" button.  If I tap on that, I can open the menu:

![image](https://cloud.githubusercontent.com/assets/928045/11189816/3a9cc65c-8c60-11e5-81e8-5851fa69d96f.png)

Do you see this on your Nexus 9?

---

**Comment by kalon33** [comment]
![tmp_27971-screenshot_20151117-144908849507548](https://cloud.githubusercontent.com/assets/460731/11213107/8b2a1cd4-8d3a-11e5-8684-ab9a39d54531.png)

**Comment by barbeau** [comment]
@kalon33 Strange - do you see the same thing if you change the orientation?

**Comment by kalon33** [comment]
Yes, unfortunately...
![tmp_27971-screenshot_20151117-164411-297520466](https://cloud.githubusercontent.com/assets/460731/11216255/8eb15c22-8d4a-11e5-8aef-39b6039fa234.png)

**Comment by barbeau** [comment]
That's bizarre.  Maybe it's something with layouts and compatibility modes for large screen devices.

At any rate, we can probably just stick a "Settings" entry at the bottom of the left sliding panel - that would be a reasonable solution for devices that don't have the compatibility mode menu button.

**Comment by kalon33** [comment]
@barbeau  sure, I think that would be a good solution. 

OffTopic questions: Is this still possible to add OTP servers to the app? Do you have an idea on how to fix geocoding problem #396 ?

Thanks :)

**Comment by barbeau** [comment]
@kalon33 Yes, still possible to add servers to the app - just send me the server info.  Haven't had a chance to look at issue at bottom of #396 yet.

**Comment by kalon33** [comment]
@barbeau I sent you these infos in a mail a few hours ago :)  thanks for working on fixing that menu issue :)

**Comment by barbeau** [comment]
I confirmed that the "3 dots" compatibility menu does show up on a real Nexus 5 device with Android 6.0 also.  So, this problem must be unique to Nexus 9.
